code/datums/martial/krav_maga.dm

```diff
@@ -0,0 +1,171 @@
+/datum/martial_art/krav_maga//experimental
+	name = "Krav Maga"
+	var/datum/action/neck_chop/neckchop = new/datum/action/neck_chop()
+	var/datum/action/leg_sweep/legsweep = new/datum/action/leg_sweep()
+	var/datum/action/lung_punch/lungpunch = new/datum/action/lung_punch()
+
+/datum/action/neck_chop
+	name = "Neck Chop - Injures the neck, stopping the victim from speaking for a while."
+	button_icon_state = "neckchop"
+
+/datum/action/neck_chop/Trigger()
+	if(owner.incapacitated())
+		owner << "<span class='warning'>You can't use Krav Maga while you're incapacitated.</span>"
+		return
+	owner << "<b><i>Your next attack will be a Neck Chop.</i></b>"
+	owner.visible_message("<span class='danger'>[owner] assumes the Neck Chop stance!</span>")
+	var/mob/living/carbon/human/H = owner
+	H.martial_art.streak = "neck_chop"
+
+/datum/action/leg_sweep
+	name = "Leg Sweep - Trips the victim, rendering them prone and unable to move for a short time."
+	button_icon_state = "legsweep"
+
+/datum/action/leg_sweep/Trigger()
+	if(owner.incapacitated())
+		owner << "<span class='warning'>You can't use Krav Maga while you're incapacitated.</span>"
+		return
+	owner << "<b><i>Your next attack will be a Leg Sweep.</i></b>"
```

Comment:
These can be part of the below visible message(s) by having them as a second argument, ie `owner.visible_message("<span class='danger'>[owner] assumes the Leg Sweep stance!</span>", "<b><i>Your next attack will be a Leg Sweep.</i></b>")`